UI_UX/Frontend/src/components/ExchangeRate.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Loading } from 'carbon-components-react';
+
+//value = -1 => loading state
+export const ExchangeRate = ({ value }) => {
+  let loadingIcon = undefined;
+  if (value == -1)
+    loadingIcon = (
+      <Loading
+        className="AppHeader_exchange_rate_loading"

```

Comment:
Same, status are modifiers, so in this case:

`AppHeader` is the block.
`__exchange-rate` is the element
`--is-loading` is the modifier

Reply:
fixed